src/Cdn/custom/Update-AzFrontDoorCdnProfile.ps1

```diff
@@ -179,37 +179,51 @@ function Update-AzFrontDoorCdnProfile {
     process {
         $hasTag = $PSBoundParameters.Remove('Tag')
         $hasOriginResponseTimeout = $PSBoundParameters.Remove('OriginResponseTimeoutSecond')
+        $hasIdentityType = $PSBoundParameters.Remove('IdentityType')
+        $hasIdentityUserAssignedIdentity = $PSBoundParameters.Remove('IdentityUserAssignedIdentity')
```

Comment:
Please specify `IdentityUserAssignedIdentity` and `OriginResponseTimeoutSecond` in Update-AzFrontDoorCdnProfile.Tests.ps1

Reply:
`OriginResponseTimeoutSecond` added, `IdentityUserAssignedIdentity ` will be added into Live test in main branch.